pkg/proc/breakpoints.go

```diff
@@ -285,7 +285,7 @@ func (t *Target) SetBreakpoint(addr uint64, kind BreakpointKind, cond ast.Expr)
 	}
 
 	if kind != UserBreakpoint {
-		bpmap.internalBreakpointIDCounter++
+		// TODO: Delete this line -> bpmap.internalBreakpointIDCounter++
```

Comment:
If this just needs to be deleted the comment should be removed too. But what's the problem?

Reply:
While I was doing my tests, that line created some issues. I just tested it again and everything works so I'll remove my change.